source/lib/constants.js

```diff
@@ -1,4 +1,7 @@
 // @flow
 
+const {version} = require('../../package.json')
```

Comment:
import {version} from '../../package.json' ?

Reply:
This is at least what I have on the redux settings PR.

Reply:
I'm always of a split mind when importing JSON files. They don't have exports, and it's not specified how it works. 

So I kinda like to require them instead, because that's better specified. 

But I could change it here and then do an import/require swap later one.

Reply:
If this is your preference, we'll keep it.